.github/workflows/ci.yml

```
@@ -23,7 +23,17 @@ jobs:
         run: npm install --no-audit --no-fund
 
       - name: TypeScript check
-        run: npx tsc --noEmit
+        run: npm run type-check
+
+      - name: Run tests
+        run: npm run test:ci
+
+      - name: Upload coverage reports
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)